.gitignore

```diff
@@ -13,3 +13,4 @@ tutorials
 .prove
 /target/
 /.idea/
+cargo
```

Comment:
I'm not seeing a "cargo" file showing up anywhere.

Reply:
Oh, probably a leftover from earlier in the experiment..